doc/mob_db.txt

```diff
@@ -204,6 +204,19 @@ AttackMotion: Attack animation motion. Low value means monster's attack will be
 
 ---------------------------------------
 
+ClientAttackMotion: The time from the start of a normal attack until the damage frame shows on client. At the same time you also get stopped on the client.
+This value is only needed if you use the "synchronize_damage" feature (battle/client.conf).
+If you created a custom sprite, you want to set this value to the timing of the damage frame in your *.act file.
+
+Background:
+When a clif_damage packet is sent to the client it will also send "sdelay" (AttackMotion) as value.
+The client will ignore all values above 432, and 432 stands for 1.0x animation speed. 216 for example means play the animation at double the speed.
```

Comment:
I know what you are trying to say, but reading this explanation here gives the impression that I should set ClientAttackMotion to a maximum of 432 and if I want to double its speed I should set it to 216.
But in reality you are talking about something totally different.
Please consider moving this (useful) documentation into clif, as it may be confusing for "normal" users here.

Reply:
You want documentation to be in the source file rather than the documentation folder?

Reply:
Well not 100% sure, but since it has nothing to do with "to what should i set that value" I was thinking that may be the better place.

Reply:
I improved the doc to focus on how to set the value rather than how it works internally here.